src/containers/container_manager_client.py

```diff
@@ -216,6 +218,48 @@ def server_halt(self) -> None:
         sock.send(b"HALT")
         sock.close()
 
+    def update(self) -> None:
+        """
+        Searches for updates and installs them if needed
+        """
+        # Search for latest release
+        g = Github()
+        repo = g.get_repo('Kippiii/jabberwocky-container-manager')
+        latest = repo.get_latest_release()
+
+        # Compare release versions
+        latest_version_num = latest.title.strip()
+        if latest_version_num == VERSION:
+            pass # TODO
+
+        # Download release
+        try:
+            os = get_os()
+        except ValueError:
+            pass # TODO
+        match os:
+            case OS.WINDOWS:
+                search_for = '.exe'
+            case OS.MACOS:
+                search_for = 'darwin'
+            case OS.LINUX:
+                search_for = 'linux'
+            case _:
+                pass
```

Comment:
What happens here if the OS isn't one of these? Shouldn't this raise an exception?